checker/manual/external-tools.tex

```diff
@@ -324,60 +324,90 @@ \section{Maven\label{maven}}
             <artifactId>checker-qual</artifactId>
             <version>\ReleaseVersion{}</version>
         </dependency>
-
+        <dependency>
+            <groupId>org.checkerframework</groupId>
+            <artifactId>checker</artifactId>
+            <version>\ReleaseVersion{}</version>
+        </dependency>
+        <!-- The annotated JDK to use (change to jdk7 if using Java 7) -->
```

Comment:
Please remove the comment about jdk7 as the instructions do not yet work for JDK7.